core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1224,9 +1222,7 @@ protected void workerRegisterStart(WorkerRegisterContext context,
     processWorkerAddedBlocks(workerInfo, currentBlocksOnLocation);
     processWorkerOrphanedBlocks(workerInfo);
     workerInfo.addLostStorage(lostStorage);
-    if (options.hasBuildVersion()) {
-      workerInfo.setBuildVersion(options.getBuildVersion());
-    }
+    workerInfo.setBuildVersion(options.getBuildVersion());
```

Comment:
@jiacheliu3 this is for streaming register.

Reply:
I am blind...